client/consensus/src/tests.rs

```diff
@@ -716,3 +290,590 @@ async fn authorities_runtime_api_tests() {
+    let _ = sp_tracing::try_init_simple();
+    let keystore_path = tempfile::tempdir().expect("Creates keystore path");
+    let keystore = LocalKeystore::open(keystore_path.path(), None).expect("Creates keystore.");
+    let alice_public = keystore
+        .sr25519_generate_new(NIMBUS_KEY_ID, Some(&Keyring::Alice.to_seed()))
+        .expect("Key should be created");
+
+    // Copy of the keystore needed for tanssi_claim_slot()
+    let keystore_copy = LocalKeystore::open(keystore_path.path(), None).expect("Copies keystore.");
+    keystore_copy
+        .sr25519_generate_new(NIMBUS_KEY_ID, Some(&Keyring::Alice.to_seed()))
+        .expect("Key should be copied");
+
+    let builder = TestClientBuilder::new();
+    let backend = builder.backend();
+    let client = Arc::new(builder.build());
+    let environ = DummyFactory(client.clone());
+    let relay_client = RelayChain {
+        client: client.clone(),
+        block_import_iterations: 1u32,
+    };
+    let spawner = sp_core::testing::TaskExecutor::new();
+    let orchestrator_tx_pool = sc_transaction_pool::BasicPool::new_full(
+        Default::default(),
+        true.into(),
+        None,
+        spawner.clone(),
+        client.clone(),
+    );
+    let mock_runtime_api = MockRuntimeApi::new(Some(1000u32.into()));
+
+    let (overseer, handle) = dummy_overseer_builder(spawner.clone(), MockSupportsParachains, None)
+        .unwrap()
+        .replace_runtime_api(|_| mock_runtime_api)
+        .build()
+        .unwrap();
+    spawner.spawn("overseer", None, overseer.run().then(|_| async {}).boxed());
+
+    let min_slot_freq = 4u32;
+
+    // Build the collator
+    let params = LookAheadParams {
+        create_inherent_data_providers: move |_block_hash, _| async move {
+            let slot = InherentDataProvider::from_timestamp_and_slot_duration(
+                Timestamp::current(),
+                SlotDuration::from_millis(SLOT_DURATION_MS),
+            );
+
+            Ok((slot,))
+        },
+        block_import: environ.0.clone(),
+        relay_client: relay_client.clone(),
+        keystore: keystore.into(),
+        para_id: 1000.into(),
+        proposer: ConsensusProposer::new(environ.clone()),
+        collator_service: CollatorService::new(
+            client.clone(),
+            Arc::new(spawner.clone()),
+            Arc::new(move |_, _| {}),
+            Arc::new(environ.clone()),
+        ),
+        authoring_duration: Duration::from_millis(500),
+        cancellation_token: CancellationToken::new(),
+        code_hash_provider: DummyCodeHashProvider,
+        collator_key: CollatorPair::generate().0,
+        force_authoring: false,
+        get_orchestrator_aux_data: move |_block_hash, _extra| async move {
+            let aux_data = OrchestratorAuraWorkerAuxData {
+                authorities: vec![alice_public.into()],
+                // This is the orchestrator consensus, it does not have a slot frequency
```

Comment:
I think this comment isn't correct for this test

Reply:
fixed in [82242eb](https://github.com/moondance-labs/tanssi/pull/662/commits/82242eb5310be2e51fad9bb311e231f63c8199bc)